dali/test/python/operator/test_remap.py

```diff
@@ -148,12 +174,14 @@ def _compare_pipelines_pixelwise(self, pipe1, pipe2, N_iterations, eps=.01):
                         f"Test failed. Actual error: {noutliers / size}, expected: {eps}.")
 
     @staticmethod
-    def _measure_time(func, n_iterations=1000):
-        start = time.time()
+    def _measure_time(func, n_iterations=30):
+        times = []
         for _ in range(n_iterations):
+            start = time.perf_counter()
             func()
-        stop = time.time()
-        return stop - start
+            stop = time.perf_counter()
+        times.append(stop - start)
```

Comment:
shouldn't you append *inside* the loop? Otherwise you are only counting the last iteration. And the mean is just that one value